user-service/src/main/resources/application.yaml

```
@@ -1,9 +1,17 @@
+
 server:
-    port: 8083
+  port: 8083
 eureka:
-    client:
-        service-url:
-            defaultZone: http://localhost:8761/eureka/
\ No newline at end of file
+  client:
+    service-url:
+      defaultZone: http://localhost:8761/eureka/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)